HW3/big_text.py

```diff
@@ -0,0 +1,41 @@
+from string import punctuation
+b_text = "Python also includes a data type for sets. A set is an unordered collection with"\
+    " no duplicate elements. Basic uses include membership testing and eliminating duplicate"\
+    " entries. Set objects also support mathematical operations like union, intersection,"\
+    " difference, and symmetric difference.\n"\
+    "Curly braces or the set() function can be used to create sets. Note: to create an empty"\
+    "set you have to use set(), not {}; the latter creates an empty dictionary, a data "\
+    "structure that we discuss in the next section."
+list_punct = []
+for i in range(len(punctuation)):
+    list_punct.append(punctuation[i])
```

Comment:
Во-первых, зачем вам переводить строку с пунктуацией в список пунктуации? В цикле в 12 строке можно было проходиться напрямую по строке `punctuation`. Во-вторых, зачем это делать таким образом, когда просто можно распаковать строку в литерал.
```suggestion
list_punct = [*punctuation]
```